build.yaml

```diff
@@ -317,6 +344,25 @@ common_cli:
     # and build a rpm/deb package using docker
     parse_logs \$venv/bin/python admin/build-package --destination-path repo --distribution \${DISTRIBUTION_NAME} /flocker/dist/Flocker-\${FLOCKER_VERSION}.tar.gz
 
+  build_homebrew_package: &build_homebrew_package  |
+    # and build a homebrew
+    SDIST_URL=file://\${PWD}/dist/Flocker-\${FLOCKER_VERSION}.tar.gz
+    RECIPE_FILE=\${PWD}/Flocker\${GIT_COMMIT}.rb
+    \${venv}/bin/python -m admin.homebrew --flocker-version \${GIT_COMMIT} --sdist \${SDIST_URL} --output-file \${RECIPE_FILE}
+    \$venv/bin/python admin/build-package --destination-path repo --distribution \${DISTRIBUTION_NAME} /flocker/dist/Flocker-\${FLOCKER_VERSION}.tar.gz
+
+  install_homebrew_package: &install_homebrew_package
+    # install the new freshly baked package
+    brew update
```

Comment:
Link to an issue (which should be a follow up to this) to use the existing homebrew testing task, instead of hardcoding these instructions here.